docs/source/developer/contributing.rst

```diff
@@ -975,8 +984,8 @@ The majority of names are written in lower case. These names include:
 -  cell inspector
 -  code console
```

Comment:
Is there an indentation mismatch here?

Reply:
Is there? I see each line starting with `-  ` (hyphen and two spaces)

Reply:
I checked out the file and it looks fine. Looks good.